soil-moisture-monitor.ino

```diff
@@ -105,6 +117,20 @@ void printProgressBar(int percentage) {
 }
 
 /*
+ * Prints current temperature and humidity on LCD second line 
+ */
+void printTempAndHumidity(float temp, float humidity) {
+  lcd.setCursor(0,2);
```

Comment:
Maybe it would be good to handle communication errors here, like `if (temp < 0 || humidity < 0) return;`